processor/transformprocessor/README.md

```diff
@@ -1,10 +1,11 @@
 # Transform Processor
 
 | Status                   |                       |
-| ------------------------ | --------------------- |
+|--------------------------|-----------------------|
 | Stability                | [alpha]               |
 | Supported pipeline types | traces, metrics, logs |
 | Distributions            | [contrib]             |
+| Warnings                 | [Many](#warnings)     |
```

Comment:
This row does not feel like is part of the 'status' of the component, so maybe the table name should change? (But then again, I guess you can make the same argument for the 'Distributions' row)

Reply:
Agree that the table is very useful. I will try to think of a better name, but we can deal with this elsewhere